codeHF/workflows.yml

```diff
@@ -149,6 +151,7 @@ workflows:
 
   o2-analysis-hf-candidate-selector-lc:
     <<: *selector_3prong
+    dependencies: o2-analysis-pid-bayes
```

Comment:
This doesn't work as it makes `pid-bayes` the only dependency, which is not correct.